xml/cl.xml

```diff
@@ -7194,6 +7211,18 @@ server's OpenCL/api-docs repository.
                 <command name="clSetContentSizeBufferPoCL"/>
             </require>
         </extension>
+        <extension name="cl_ext_buffer_device_address" supported="opencl">
```

Comment:
This fails with the latest state of the `main` branch:

> element "extension" missing required attribute "revision"

Reply:
```suggestion
        <extension name="cl_ext_buffer_device_address" revision="0.1.0" supported="opencl">
```

Reply:
([from .asciidoc spec](https://github.com/KhronosGroup/OpenCL-Docs/pull/1159/files#diff-cd49946e4c7aaedc4d5adc5bd2f624d351c550d06177b1c66ad48a04b6c4196f)

Reply:
And with this small fix, XML works, but .asciidoc fails just like [here](https://github.com/KhronosGroup/OpenCL-Docs/pull/1171#issuecomment-2308994847), but seemingly on a later call:
https://github.com/SunSerega/OpenCL-Docs/actions/runs/10550058760/job/29225797422#step:8:126